fingerprint/infra/scanner/src/main/java/com/simprints/fingerprint/infra/scanner/v1/MessageDispatcher.java

```diff
@@ -111,7 +111,7 @@ private Message sendRecv(@NonNull Message request, @NonNull MESSAGE_TYPE answerT
             try {
                 pendingRequest.lock.wait(timeOutMs);
             } catch (InterruptedException e) {
-                Simber.i("Message response timed out", e);
+                Simber.Companion.i("Message response timed out", e);
```

Comment:
Not very excited about this pattern

Reply:
This is only forced in the java files (of which we have 1 or 2), in kotlin you can do `Simber.i()` directly.